core/src/main/java/com/ritense/valtimo/security/jwt/exception/TokenAuthenticatorNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.ritense.valtimo.security.jwt.exception;
```

Comment:
The build pipeline is failing because this new class is missing the Copyright information
```
[ant:checkstyle] [ERROR] /tmp/jenkins-377d2bce/workspace/ltimo_-_Backend_libraries_PR-377/core/src/main/java/com/ritense/valtimo/security/jwt/exception/TokenAuthenticatorNotFoundException.java:1: Missing a header - not enough lines in file. [RegexpHeader]
```

Reply:
Thanks, I was not aware of this . I will add the copyright header.